backend/entities/Parcel.cs

```diff
@@ -60,19 +55,19 @@ public class Parcel : Property
         /// get/set - A collection of buildings on this parcel.
         /// </summary>
         /// <typeparam name="Building"></typeparam>
-        public ICollection<Building> Buildings { get; private set; } = new List<Building>();
+        public ICollection<ParcelBuilding> Buildings { get; } = new List<ParcelBuilding>();
```

Comment:
maybe ParcelBuildings?

Reply:
Some would recommend that because of the object, but it would not be the normal naming convention for a relationship.  Due to .net core entity framework not automatically handling many-to-many relationships it requires us to manually perform them.  I still prefer the mental relationship naming convention which has been the practice for many years now with ORMs.  However I do understand it is odd since the object doesn't match 100%.